imjoy/runners/subprocess.py

```diff
@@ -24,10 +24,11 @@
     run_process,
 )
 
-DEFAULT_REQUIREMENTS_PY2 = ["numpy", "python-socketio[client]"]
-DEFAULT_REQUIREMENTS_PY3 = ["numpy", "python-socketio[client]", "janus"]
-REQ_PSUTIL = ["psutil"]
-REQ_PSUTIL_CONDA = ["conda:psutil"]
+DEFAULT_REQUIREMENTS = ["numpy==1.16.4", "python-socketio[client]==4.0.3"]
```

Comment:
I think we should leave the `numpy` here, since it's not really important, and plugins will need different versions of it.